dotCMS/src/main/webapp/html/portlet/ext/contentlet/publishing/view_publish_tool.jsp

```diff
@@ -287,53 +286,48 @@
 
 <div class="portlet-main">
     <div id="mainTabContainer" dojoType="dijit.layout.TabContainer" dolayout="false">
-
-        <div id="queue" dojoType="dijit.layout.ContentPane" title="<%= LanguageUtil.get(pageContext, "publisher_Queue") %>" >
+        
+        <div id="audit" dojoType="dijit.layout.ContentPane" title="<%= LanguageUtil.get(pageContext, "publisher_Audit") %>" >
             <div class="portlet-toolbar">
-				<div class="portlet-toolbar__actions-primary">
-                    
-                </div>
-                <div class="portlet-toolbar__actions-secondary">
+				<div class="portlet-toolbar__actions-primary"></div>
```

Comment:
Codacy found an issue: [Do not use an attribute called class.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6047717)